routes.js

```diff
@@ -69,7 +68,7 @@ module.exports = (app) => {
   // All routes below are authenticated
   router.use(passport.authenticate("jwt", { session: false }));
   router.get("/restaurants/me", controller.getRestaurant);
-  router.put("/restaurants/me", controller.updateRestaurant);
+  router.put("/restaurants/:id", controller.updateRestaurant);
```

Comment:
What's the motivation for this change?

Reply:
Okay that does sounds weird but it would be weirder if there are requests that can change other restaurants/users... But let's get it merged for now.